From abec5f985d47d87171a3bfde91cdede960e6945c Mon Sep 17 00:00:00 2001
From: BastianZim <10774221+BastianZim@users.noreply.github.com>
Date: Sun, 4 Dec 2022 15:59:50 -0500
Subject: [PATCH] Make numpy Conda-Forge compatible

---
 pyproject.toml | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 7c3d57b..0c29925 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -30,10 +30,7 @@ python = ">=3.7.1, <4.0"
 boto3 = "^1.24.11"
 botocore = "^1.27.11"
 pandas = "^1.2.0, <=1.5.1, !=1.5.0" # Exclusion per: https://github.com/aws/aws-sdk-pandas/issues/1678
-numpy = [
-    { version = ">=1.21.0 <1.23.4", markers = "python_full_version >= '3.7.1' and python_full_version < '3.11.0'" },
-    { version = "^1.23.4", markers = "python_full_version >= '3.11.0'" },
-]
+numpy = ">=1.21.0,<2"
 pyarrow = ">=2.0.0,<10.1.0"
 redshift-connector = "~2.0.889"
 pymysql = "^1.0.0"
-- 
2.38.1

